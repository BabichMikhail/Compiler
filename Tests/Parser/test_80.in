a[10] := b^;