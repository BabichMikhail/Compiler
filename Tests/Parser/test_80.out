   ^
      b
:=
      10
   []
      a
